packages/kamino-sdk/tests/stratCreation.test.ts

```diff
@@ -192,7 +195,13 @@ describe('Kamino strategy creation SDK Tests', () => {
     console.log('setup strategy fees tx hash', txHash);
 
     // after strategy creation we have to set the reward mappings so it autocompounds
-    let updateRewardMappingIxs = kamino.getUpdateRewardsIxs(signer.publicKey, newStrategy.publicKey);
+    let updateRewardMappingIxs = await kamino.getUpdateRewardsIxs(signer.publicKey, newStrategy.publicKey);
+    const updateRewardMappingTx = await kamino.getTransactionV2Message(signer.publicKey, updateRewardMappingIxs);
+    const updateRewardMappingsTransactionV0 = new VersionedTransaction(updateRewardMappingTx);
+    updateRewardMappingsTransactionV0.sign([signer]);
+    //@ts-ignore
+    txHash = await sendAndConfirmTransaction(kamino._connection, updateRewardMappingsTransactionV0);
```

Comment:
You can just use connection variable instantiated in the tests instead of having to use @ts-ignore and accessing a private property

Reply:
the `sendAndConfirmTransaction` interface is the issue, not the connection. It is expecting other interfaces and seems like the version we have doesn't "support" transactionV2